test/newsTest.js

```diff
@@ -0,0 +1,24 @@
+describe("Note Class", () => {
+  var news = new News();
+  it("is defined", () => {
+    checkIf(news).isClass();
+  })
+  describe("getDate()", () => {
+    it("is defined", () => {
+      checkIf(news).has("getDate").function();
+    })
```

Comment:
this test is redundant given that the test below would also fail if the function didn't exist.